### DISCUSSION TITLE
Filter issues on the beta boards by clicking on the repository name

### DISCUSSION THREAD

**Discussion by leitneratselerity** [community/community#9221] [Projects and Issues]
On the Beta Projects, when in the Board view, allow us to click on the name of a repository, to filter all issues from that repo only.
I.e., provide a similar behavior as when clicking on a tag pill or a milestone pill, but for repository names (which is found at the top of the issue card).

**Comment by JMC-PK** [comment]
+1 i just came to find an issue like this. I want to be able to choose all, some, or even 1 repo in my org with lots of repos.

the table view is nice because i can see which issues are in which repo, but i have to hunt and seek to find the same issue between board and table view.
